etc/deploy/compose/compose.yaml

```diff
@@ -10,3 +10,5 @@ services:
     shm_size: '1g'
     command: >
       postgres -c random_page_cost=1.1 -c max_parallel_workers_per_gather=4 -c shared_preload_libraries='pg_stat_statements'
+#    volumes:
```

Comment:
what is this for?

Reply:
For starting the container with a database dump.

Reply:
And is it fine to keep it commented or should it be executed?